src/main/webapp/js/individual-listing-loader.js

```diff
@@ -34,29 +33,81 @@ function fetchListings() {
  * @return {Element}
  */
 function buildListingsDiv(listing) {
-  const headerDiv = document.createElement("div");
-  headerDiv.classList.add("message-header");
-  headerDiv.classList.add("padded");
-  headerDiv.classList.add("border-bottom");
+  
+  /* contains image */
+  const pictureDiv = document.createElement("img");
+  pictureDiv.classList.add("col-lg-6");
+  pictureDiv.classList.add("col-md-6");
+  pictureDiv.classList.add("col-sm");
+  pictureDiv.src = "http://newnation.sg/wp-content/uploads/random-pic-internet-07.jpg";
```

Comment:
need to figure out how to get the picture off the blobstore, this is a placeholder for now 👍